README.md

```diff
@@ -14,6 +14,12 @@ While producing synthetic data with [metasyn](https://github.com/sodascience/met
 > Currently, the disclosure control plugin is work in progress. Especially in light of this, we disclaim
 any responsibility as a result of using this plugin. 
 
+## Installing the plugin
+
+1. **Clone the Repository**: Click the green `code` button at the top right of this page, and follow the instructions from there.
+
+2. **Install the Plugin**: In your terminal (with the Python environment that you want to install to active) run the following command: ``pip install <path to cloned repository>``. Or alternatively, to install it from a notebook: ``!pip install <path to cloned repository>``.
```

Comment:
`pip install git+https://github.com/sodascience/metasyn-disclosure-control.git` is an easier option

Reply:
<img width="514" alt="image" src="https://github.com/sodascience/metasyn-disclosure-control/assets/22395533/8816f06c-b1a6-4beb-9844-e0610c0ff1a2">